LiskHQ/lisk-nano
# Improve clarity of second passphrase registration texts - Closes #1026

### What was the problem?
Fee for the second passphrase was not clear enough to some users.

### How did I fix it?
Made the fee more visible and improved the text description.

### How to test it?
Try to register a second passphrase. Check also fee in Vote and Delegate registration.

### Review checklist
- The PR solves #1026
- All new code is covered with unit tests
- All new code follows best practices
